Dnd.Ddd.Infrastructure/Middleware/UserTypes/CharacterStateType.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Dnd.Ddd.Infrastructure.Database.Middleware.UserTypes.EnumBase;
+using Dnd.Ddd.Model.Character.CharacterStates;
+using Dnd.Ddd.Model.Character.CharacterStates.Contract;
+
+namespace Dnd.Ddd.Infrastructure.Database.Middleware.UserTypes
+{
+    internal class CharacterStateType : StringEnumType<CharacterStates, CharacterState>
+    {
+        public override bool IsMutable => false;
```

Comment:
Property should be placed after class constructors.

Reply:
changed